baywatch.install

```diff
@@ -42,3 +45,14 @@ function baywatch_import_sdpa_password_policy() {
 function baywatch_update_8001() {
   baywatch_import_sdpa_password_policy();
 }
+
+/**
```

Comment:
Are we sure this is the case? I have a feeling this means it will send nothing now.

Reply:
The default value shipped with the module is an empty string - https://git.drupalcode.org/project/queue_mail/-/blob/8.x-1.x/config/install/queue_mail.settings.yml#L1

Reply:
Yep, but that could just mean the default is to do nothing :). https://git.drupalcode.org/project/queue_mail/-/blob/8.x-1.x/queue_mail.module#L30 is probably relevant... will the path matcher match on null or does it need an `*`.

Reply:
Yeah you're right, thanks eagle-eye!!

Reply:
FYI I've opened an issue to change the default value in queue_mail. https://www.drupal.org/project/queue_mail/issues/3153119